sklearn/utils/__init__.py

```diff
@@ -273,7 +273,8 @@ def _determine_key_type(key, accept_slice=True):
 # TODO: remove in 0.24
 @deprecated("safe_indexing is deprecated in version "
             "0.22 and will be removed in version 0.24.")
-def safe_indexing(X, indices, axis=0):
+@_deprecate_positional_args
+def safe_indexing(X, indices, *, axis=0):
```

Comment:
we're deprecating this func anyway.

Reply:
yeah I already have a PR that removes it